packages/zowe-explorer-api/src/profiles/ZoweExplorerZosmfApi.ts

```diff
@@ -11,6 +11,7 @@
 
 import * as zowe from "@zowe/cli";
 import { ZoweExplorerApi } from "./ZoweExplorerApi";
+import { FileAttributes, permStringToOctal } from "../utils";
```

Comment:
Should we try to make new imports as specific as possible?
`import { FileAttributes, permStringToOctal } from "../utils/files";`